internal/service/synthetics/canary.go

```diff
@@ -385,7 +390,13 @@ func resourceCanaryRead(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("error setting vpc config: %w", err)
 	}
 
-	if err := d.Set("run_config", flattenCanaryRunConfig(canary.RunConfig)); err != nil {
+	// get environment variables since they a
```

Comment:
maybe this was intended to describe how we get the vars from the configuration since `GetCanary` does not return them for us?

Reply:
comment from a thought that I didn't complete 😄 .